net.crepererum.breitbandmessung.yml

```diff
@@ -0,0 +1,42 @@
+---
+app-id: net.crepererum.breitbandmessung
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+tags:
+  - proprietary
+command: breitbandmessung
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=x11
+modules:
+  - name: breitbandmessung
+    buildsystem: simple
+    build-commands:
+      - '. ${FLATPAK_BUILDER_BUILDDIR}/build.bash'
```

Comment:
it would be better to have the build-commands inline.

Reply:
yes. that's how it's done.

Reply:
done